perfil/mandate/management/commands/load_parties_filiation.py

```diff
@@ -0,0 +1,33 @@
+from perfil.utils.management.commands import ImportCsvCommand
+
+from perfil.election.management.commands import (party_keys,
+                                                 person_keys_birthdate)
+from perfil.mandate.models import PartyFiliation
+from perfil.party.models import Party
+from perfil.person.models import Person
+
+
+class Command(ImportCsvCommand):
+
+    to_cache = ((Person, person_keys_birthdate), (Party, party_keys))
+    model = PartyFiliation
+    bulk_size = 2 ** 4
```

Comment:
Only 16? Is there a reason for such a small bulk?

Reply:
I didn't see this. Going back to original